spec/taxon_breadcrumbs_spec.rb

```diff
@@ -91,7 +74,19 @@ def breadcrumbs_for(content_item)
 
   def content_item_with_parents(parents)
     {
-        "links" => { "parent_taxons" => parents }
+      "links" => {
```

Comment:
As discussed IRL, I think it would be beneficial to wrap the test cases around 2 contexts:

```
context "given a content item which is a taxon with parent taxons" do
   # In here, we use the link type `parent_taxon` and make sure it works
end

context "given a content item tagged to a taxon" do
   # In here, we use the link type `taxons` and make sure it works as well
end
```

This way it's clearer to the reader that the `taxon_breadcrumbs` generates data for both the Taxon navigation pages and also for the content items that represent content pages.